hugr/src/extension/op_def.rs

```diff
@@ -265,11 +265,19 @@ impl Debug for SignatureFunc {
 /// Different ways that an [OpDef] can lower operation nodes i.e. provide a Hugr
 /// that implements the operation using a set of other extensions.
 #[derive(serde::Deserialize, serde::Serialize)]
+#[serde(untagged)]
 pub enum LowerFunc {
     /// Lowering to a fixed Hugr. Since this cannot depend upon the [TypeArg]s,
     /// this will generally only be applicable if the [OpDef] has no [TypeParam]s.
-    #[serde(rename = "hugr")]
-    FixedHugr(ExtensionSet, Hugr),
+    FixedHugr {
+        /// The extensions required by the [`Hugr`]
```

Comment:
Hmmmm, perhaps to think about in the future, but if we insist that the `Hugr` is DFG-rooted, then we can extract the ExtensionSet from the root node's `extension_reqs`, so I'm not sure we really need this...